gclient/binaries/build.rs

```diff
@@ -0,0 +1,21 @@
+// This file is part of Gear.
+//
+// Copyright (C) 2023 Gear Technologies Inc.
```

Comment:
I suggest removing the year number from the copyright since it is not consistent across the source code and we will not update it later

Reply:
I see most of files have year when just a few don't